backend/src/main/java/ca/bc/gov/hlth/mohums/security/SecurityConfig.java

```diff
@@ -50,6 +53,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .mvcMatchers(HttpMethod.GET,"/users/{userId}/groups").access(String.format("hasRole('%s') and hasRole('%s')", viewGroupsRole, viewUsersRole))
```

Comment:
Just to nitpick but I'd say viewGroupsRole and viewUsersRole in line 53's String.format should be swapped to match the similar String.format used in line 56.